# Reto de fin de semana

## Enunciado

**Parte 1.** Crea una función que imprima por pantalla las horas, minutos y segundos de un tiempo dado en segundos.


**Parte 2.** Crea la función contraria, indicándole el número de horas, minutos y segundos y muestre el total de segundos

In [7]:
h = 3600
m = 60

def secondsToTime(totalseconds):
    hours = (totalseconds // h)
    minutes = ((totalseconds % h) // m)
    seconds = (totalseconds % m)
    return "{:02d}:{:02d}:{:02d}".format(hours, minutes, seconds)

totalseconds = int(input("Insert the time in seconds "))
print(secondsToTime(totalseconds))

01:02:41


In [8]:
def timeToSeconds(hours, minutes, seconds):
    totalseconds = hours * 3600 + minutes * 60 + seconds
    return totalseconds

hours = int(input("Enter the amount of hours "))
minutes = int(input("Enter the amount of minutes "))
seconds = int(input("Enter the amount of seconds "))


print(timeToSeconds(hours, minutes, seconds))

3761


## Ayuda a la implementación

Recuerda que un minuto tiene 60 segundos, una hora tiene 60 minutos o lo que es lo mismo 3600 segundos.

El resultado puedes devolverlo así.

```
return horas, minutos, segundos
```

# Datos de prueba
Un ejemplo de cambio de segundos a horas:minutos y segundos:
3761 -> 1:2:41
Un ejemplo del cambio contrario:
3:10:56 -> 11456